Add data and updated images

zietzm opened the pull request:
Addresses https://github.com/greenelab/xswap-manuscript/issues/64

dhimmel commented:
Nice! A couple thoughts:

On the data files, should we apply the figure name after the number like: `6C_auroc_dists.tsv` (not sure if that is an actual match). This way we're less dependent on just the figure numbers which can be unstable.

Consider prefixing with `figure_` or putting in `data/figure` if we think that will help keep the data directory organized. I envision we might want to unignore some other files in the data directory and track them, possibility using git LFS or not.

Did you want to commit the updated notebooks with the code to dump the figure data?

Go ahead and merge this PR as you see fit, I'll provide feedback but not hold up merging.

dhimmel commented:
One other thought is that Figure 1 and 3 might benefit from similar treatment. They were not identified in https://github.com/greenelab/xswap-manuscript/issues/64, but might save us time to do them now assuming it will be needed in future.

zietzm commented:
Comments addressed:
- [X] Added descriptive names to data files
- [X] Moved figure data files to the `data/figure` directory
- [X] Added updated notebooks from latest run
- [X] Added updated figures 1 and 3

Unaddressed: 
- [ ] Larger files should be included. Total size of `data/task1` is 13 GB, `data/task3` 3.8 GB, `data/feature-degree` is 1.8 GB. Will need to upload these elsewhere e.g. Zenodo
